index.html

```diff
@@ -0,0 +1,108 @@
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <header id="header">
+    <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" id="header-img"
+      alt="">
+    <nav id="nav-bar">
+      <ul>
+        <li><a class="nav-link" href="#features">Features</a></li>
+        <li><a class="nav-link" href="#how-it-works">How it works</a></li>
+        <li><a class="nav-link" href="#pricing">Pricing</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <form id="form" action="https://www.freecodecamp.com/email-submit">
+      <h1>Handcrafted, home-made masterpieces</h1>
+      <input id="email" type="email" placeholder="Enter you email address"> <br>
+      <input id="submit" type="submit" value="GET STARTED">
+    </form>
+    <section id="features">
+      <div>
+        <i class="fa fa-3x fa-fire"></i>
+        <h1>Premium Materials</h1>
+        <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your
+          purchase.</p>
+      </div>
+      <div>
+        <i class="fa fa-3x fa-truck"></i>
+        <h1>Fast Shipping</h1>
+        <p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if
+          you are
+          not satisfied.</p>
+      </div>
+      <div>
+        <i class="fa fa-3x fa-battery-full"></i>
+        <h1>Quality Assurance</h1>
+        <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the
+          pitch
+          of your
+          instrument.</p>
+      </div>
+
+    </section>
+    <section id="how-it-works">
+      <iframe id="video" height="315"
+        src="https://www.youtube-nocookie.com/embed/y8Yv4pnO7qc?rel=0&amp;controls=0&amp;showinfo=0" frameborder="0"
+        allowfullscreen></iframe>
+    </section>
+    <section id="pricing">
+      <div>
+        <div class="head">
+          <h4>TENOR TROMBONE</h4>
+        </div>
+        <h2>$600</h2>
+        <p>Lorem ipsum</p>
+        <p>Lorem ipsum</p>
+        <p>Lorem ipsum dolor</p>
+        <p>Lorem ipsum</p>
+        <button>SELECT</button>
+      </div>
+      <div>
+        <div class="head">
+          <h4>BASS TROMBONE</h4>
+        </div>
+        <h2>$900</h2>
+        <p>Lorem ipsum</p>
+        <p>Lorem ipsum</p>
+        <p>Lorem ipsum dolor</p>
+        <p>Lorem ipsum</p>
+        <button>SELECT</button>
+      </div>
+      <div>
+        <div class="head">
+          <h4>VALVE TROMBONE</h4>
+        </div>
+        <h2>$1200</h2>
+        <p>Lorem ipsum</p>
+        <p>Lorem ipsum</p>
+        <p>Lorem ipsum dolor</p>
+        <p>Lorem ipsum</p>
+        <button>SELECT</button>
+      </div>
+    </section>
+  </main>
+  <footer>
+    <a href="">Privacy</a>
+    <a href="">Terms</a>
+    <a href="">Contacts</a>
```

Comment:
I think it would be best to make use of the semantic tag `ul` for lists like this for good code readability and accessibility